src/nakamoto-signer-developer-guide.md

```diff
@@ -0,0 +1,196 @@
+# What is this
+
+Draft documentation for sBTC Signer specifically for the Nakamoto [release](https://stacks-network.github.io/sbtc-docs/sbtc-roadmap.html)
+
+The readers of this are assumed to be anyone who is looking to run a sBTC Signer with minimal custom configuration.
+
+# Disclaimer
+
+Development is on going and changes to the Signer configuration should be expected. This document will be updated every sprint to ensure its accuracy.
+
+## Prerequisites
+
+Rust - [To install](https://www.rust-lang.org/tools/install).
+
+Accessible Stacks node - A Stacks node running the stackerDB instance which is used for signer communication and transaction monitoring and broadcasting.
+
+Accessible Bitcoin node - A Bitcoin node used for transaction monitoring and broadcasting.
+
+A Stacks Address - Identify your address as a Signer
+
+## Installing
+
+### Building from Source
+
+If you wish to compile the default binary from source, follow the steps outlined below. Otherwise, download the binary directly (below)
+
+1. First, clone the Stacks sBTC mono repository:
+
+```console
+git clone git@github.com:stacks-network/stacks-blockchain.git
+```
+
+2. Next, navigate to the stacks-signer directory:
+
+```console
+cd stacks-blockchain/stacks-signer
+```
+
+3. Checkout the appropriate release branch you wish to use if you are not using the default main branch
+
+```console
+git checkout main
+```
+
+4. Compile the signer binary:  
+   Note the binary path defaults to `target/release/stacks-signer`.
+
+```console
+cargo build --release
+```
+
+### Downloading the Binary
+
+1. First, download the precompiled default [TODO:NEED:LINK](LINK).
+
+2. Untar the file
+
+```console
+tar -xvf signer_binary.tar
+```
+
+3.  Check Extracted Files:
+    After running the untar command, the contents of the tar file should be extracted to the current directory. You should see the signer binary (stacks-signer-mini) and the configuration file (signer.toml) listed among the extracted files.
```

Comment:
change stacks-signer-mini to stacks-signer

Reply:
fixed 97695ec9ce37137f9836f3e62bd22c7701fb51de